tasks/pre_checks/validate_network_test.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Validate network connectivity check configuration
+  block:
+  - debug: var=he_network_test
+  - name: Fail if he_network_test is not valid
+    fail:
+      msg: "Invalid he_network_test defined"
+    changed_when: true
+    when: he_network_test not in ['dns', 'ping', 'tcp', 'none']
+  - name: Validate TCP network connectivity check parameters
+    block:
+    - debug: var=he_tcp_t_address
+    - name: Fail if he_tcp_t_address is not defined
+      fail:
+        msg: "No he_tcp_t_address is defined"
+      changed_when: true
+      when:
+        ( he_tcp_t_address is undefined ) or
+        ( he_tcp_t_address is none ) or
+        ( not he_tcp_t_address|trim )
```

Comment:
I fear that this is going to generate a warning on ansible 2.8, see https://github.com/oVirt/ovirt-ansible-hosted-engine-setup/pull/165 for instance